factory/disabled/stakingDataProvider.go

```diff
@@ -0,0 +1,65 @@
+package disabled
+
+import (
+	"math/big"
+
+	"github.com/ElrondNetwork/elrond-go/state"
+)
+
+var emptyBI = big.NewInt(0)
```

Comment:
This name seems a bit unfortunate, can we change it to `zero`?

Reply:
renamed to `zeroBI`